src/FacebookEventForwarder.js

```diff
@@ -256,15 +256,17 @@
             }
 
             function logPageEvent(event, eventName) {
-                var params = cloneEventAttributes(event);
+                var params = createParameters(event);
+                var eventID = createEventId(event);
+
                 eventName = eventName || event.EventName;
                 if (event.EventName) {
                     params['content_name'] = event.EventName;
                 }
-                fbq('trackCustom', eventName || 'customEvent', params);
+                fbq('trackCustom', eventName || 'customEvent', params, eventID);
             }
 
-            function cloneEventAttributes(event) {
+            function createParameters(event) {

```

Comment:
Nitpick: `createParameters` sounds too vague. What kind of parameters is it creating?

Reply:
yea, i was using a FB term.  i thought it might be confusing eventattrs vs event id, but i'll revert. i think it's distinct enough